clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeleteExecutorGvcf.scala

```diff
@@ -37,7 +37,8 @@ class DeleteExecutorGvcf(deleteGvcf: DeleteGvcf) extends Executor {
               project = Some(deleteGvcf.transferGvcfV1Key.project),
               sampleAlias = Some(deleteGvcf.transferGvcfV1Key.sampleAlias),
               version = Some(deleteGvcf.transferGvcfV1Key.version)
-            )
+            ),
+            includeDeleted = false
```

Comment:
Shouldn't this parameter default to `false`?

Reply:
I got rid of the default because I wanted to be explicit here about not including deleted records. Now that the integration-tests have been ported away from using the web-client, these are the only spots that were affected by the change.